.github/workflows/blank.yml

```diff
@@ -1,5 +1,7 @@
 name: Build cross compilers
 'on':
+  pull_request:
+
```

Comment:
```suggestion
```

Reply:
I will remove this line after trying version 11.3 and then open the PR.